### ISSUE TITLE
how to reset stats

### ISSUE THREAD

**Issue by SliderKP** [Stadus/stadus_skills#1]
how to reset stats when a user die its possible? maybe a trigger event in esx ambulancejob?

**Comment by ScaredDonut** [comment]
Yes, either drop the table for said user by matching the identifier, or set all of skills to 0.  It will require some custom coding, however. You will want to do it in esx_ambulance since half the code is already done.
